templates/zerver/help/preview-your-message-before-sending.md

```diff
@@ -1,11 +1,12 @@
 # Previewing your message
 
-You can preview your message before you send it by clicking the **eye** (![eye](/static/images/help/eye.png)) icon.
+You can preview your message before you send it by clicking the **eye**
+(![eye](/static/images/help/eye.png)) icon at the bottom of the **Compose message box**.
 
-![Original content](/static/images/help/before-preview.png)
+This is bring up the message preview:
```

Comment:
In this line, "is" does not agree in tense with "bring". Something like "This will bring up the message preview" is a good option.